Client.xcodeproj/project.pbxproj

```diff
@@ -467,9 +467,12 @@
 		2FCAE2771ABB531100877008 /* SwiftData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCAE25B1ABB531100877008 /* SwiftData.swift */; };
 		2FCAE2841ABB533A00877008 /* MockFiles.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCAE2791ABB533A00877008 /* MockFiles.swift */; };
 		2FCAE33E1ABB5F1800877008 /* Storage-Bridging-Header.h in Headers */ = {isa = PBXBuildFile; fileRef = 2FCAE33D1ABB5F1800877008 /* Storage-Bridging-Header.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		2FCEED7B26EA64A400892699 /* HistoryNode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCEED7A26EA64A400892699 /* HistoryNode.swift */; };
+		2FCEED7D26EA6AF900892699 /* BookmarkNode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCEED7C26EA6AF900892699 /* BookmarkNode.swift */; };
+		2FCEED8526EAAB6C00892699 /* HistoryAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCEED8426EAAB6C00892699 /* HistoryAPI.swift */; };
+		2FCEEDC126EBDEDE00892699 /* BookmarkAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FCEEDC026EBDEDE00892699 /* BookmarkAPI.swift */; };
```

Comment:
Possibly add `Extensions` suffix to all these files (i.e. `HistoryNodeExtensions.swift`) to match other files and not confuse the fact that we aren't actually declaring these types ourselves

Reply:
Fix at https://github.com/brave/brave-ios/pull/4024/commits/fea5c9c4720906a7423c50e9a93bf0f8c1e2028b